arraycontext/impl/pytato.py

```diff
@@ -148,6 +148,28 @@ def arctan2(self, y, x):
         import pytato as pt
         return rec_multimap_array_container(pt.arctan2, y, x)
 
+    def ravel(self, a, order="C"):
+        def _rec_ravel(a):
+            import pytato as pt
+            if order in "FC":
+                return pt.reshape(a, (-1,), order=order)
+            elif order == "A":
+                if a.flags.f_contiguous:
```

Comment:
This could be rewritten as just having a fixme and return `pt.reshape(-1, order="C")`.

Reply:
Done